internal/testutil/storage_cleaner.go

```diff
@@ -165,20 +120,8 @@ func DeleteBucketIfExists(ctx context.Context, client *storage.Client, bucket st
 	return fmt.Errorf("failed to delete bucket %q", bucket)
 }
 
-// WaitForBucketToExist waits for a bucket to exist, as it can take time to propagate
-// Errors after 10 unsuccessful attempts at retrieving the bucket's attrs
-func WaitForBucketToExist(ctx context.Context, t *testing.T, b *storage.BucketHandle) {
-	t.Helper()
-	Retry(t, 10, 30*time.Second, func(r *R) {
-		if _, err := b.Attrs(ctx); err != nil {
-			// Bucket does not exist
-			r.Errorf("Bucket was not created")
-			return
-		}
-	})
-}
-
-// UniqueBucketName returns a unique name with the test prefix.
+// UniqueBucketName returns a unique name with the test prefix
+// Any bucket created with this prefix may be deleted by DeleteExpiredBuckets.
```

Comment:
Is this actually true?

If so we should be sure to leave bucket prefixes alone for existing tests so we don't accidentally try to delete any buckets that people want to save.

Reply:
Hmm, so it's true in that it "may" be lol. That is, if you call `DeleteExpiredBuckets` with the same prefix, which we do in storage tests I believe, those buckets will be deleted. But it's confusing, so I'll delete this comment. We definitely aren't storing prefixes and then deleting buckets based on that.

Reply:
Could you double check that you haven't changed prefixes for anything that does use DeleteExpiredBuckets? Otherwise we are good I'd say.

Reply:
Check done - nothing to note.